timely_beliefs/beliefs/probabilistic_utils.py

```diff
@@ -401,17 +401,25 @@ def set_truth(
 
 def calculate_crps(df: "classes.BeliefsDataFrame") -> "classes.BeliefsDataFrame":
     """Compute the continuous ranked probability score for a BeliefsDataFrame with a probabilistic (or deterministic)
-    forecast and observation. This function supports a probabilistic observation, too."""
+    forecast (event_value column) and observation (reference_value column).
+    This function supports a probabilistic observation, too.
+
+    References
+    ----------
+    Hans Hersbach. Decomposition of the Continuous Ranked Probability Score for Ensemble Prediction Systems
```

Comment:
It would be neat if we can list literature we use in this package in the main Readme (as well), to show prominently that we use/build on scientific knowledge.

Reply:
Made this a separate issue. #7